sig-services/service-requirements/customer-zero.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Have a customer zero
+persona: Service developer
+phase: Service development
+---
+
+**This requirement is optional**
+
+Having customer zero allows service to get instant feedback before the public release. From customer zero, developers can gather feedback on features of their service, UX design, and others. An important part of this process is not to share any insider information about the project with customer zero. Customer should be allowed to use the service on their own but should be watched by developers and in close contact with support.
+
+## Requirements
+
+- Customer zero is established; customer zero should not have any insider knowledge of the project and should be allowed to use the service on their own
```

Comment:
```suggestion
- Customer zero should be identified
- Customer zero should not have any developer or maintainer knowledge of the project and should be allowed to use the service on their own
```

Reply:
Changed in latest commit good point :+1: